### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by rita-mazets**

### CODE REVIEW

File: lab3/index.html [hunk 1]
```diff
@@ -0,0 +1,2406 @@
...
+						</div>
+						<div class="calendar-ten">
+							<a href="#accesmant"><button type="button"class="colendar-button colendar-button-active"></button></a>
+							<a href="#accesmant"><button type="button"class="colendar-button colendar-button-active"></button></a>
+						</div>
+					</div>
+					<div class="row">
+						<div class="calendar-ten">
+							<a href="#accesmant"><button type="button"class="colendar-button colendar-button-active"></button></a>
+							<a href="#accesmant"><button type="button"class="colendar-button colendar-button-active"></button></a>
```
Comment (nastik1811) [inline lab3/index.html]: It's invalid html. You can't place interactive elements inside a link. You can style `a` itself however you like, no need in button element

Comment (nastik1811) [reply lab3/index.html]: Also, you probably have a typo in the href  :)

File: lab3/index.html [hunk 2]
```diff
@@ -0,0 +1,2406 @@
...
+					<option value="0" id="mood" disabled>Mood</option>
+					<option value="1">Well</option>
+					<option value="2">Not bad</option>
+					<option value="3">Terrible</option>
+				</select>
+			</div>
+			<div class="small-container" style="margin-top: 8%;">
+				<span >Here you can describe your week</span>
+				<textarea class="accesment-textarea"></textarea>
+				<button type="submit" class="custom-button accesmant-button" style="margin-top: 5%;">Add pictures</button>
```
Comment (nastik1811) [inline lab3/index.html]: i guess it's `input type="file"`

File: lab3/index.html [hunk 3]
```diff
@@ -0,0 +1,2406 @@
...
+						<div class="calendar-ten">
+							<a href="#accesmant"><button type="button"class="colendar-button"></button></a>
+							<a href="#accesmant"><button type="button"class="colendar-button"></button></a>
+						</div>
+					</div>
+					<div class="year" style="top: 65px; color:white">35</div>
+				</section>
+			</div>
+		</div>
+		<div class="accesmant article_container">
```
Comment (nastik1811) [inline lab3/index.html:2370]: As a user use these fields to submit some information about a week, it should be a `form`

File: lab3/index.html [hunk 4]
```diff
@@ -0,0 +1,2406 @@
...
+			<div class="main-text ">
+				<a name="accesmant"></a>
```
Comment (nastik1811) [inline lab3/index.html]: What is the goal of `a` here?

File: lab3/index.html [hunk 5]
```diff
@@ -0,0 +1,2406 @@
...
+				<span>How do you rate this week?</span>
```
Comment (nastik1811) [inline lab3/index.html]: Looks more like a heading

File: lab3/index.html [hunk 6]
```diff
@@ -0,0 +1,2406 @@
...
+			<div class="sign_in" style="padding-top: 5%;">
+				<select id="sdate-month" class="form-select accesment-mood">
+					<option value="0" id="mood" disabled>Mood</option>
+					<option value="1">Well</option>
+					<option value="2">Not bad</option>
+					<option value="3">Terrible</option>
+				</select>
+			</div>
+			<div class="small-container" style="margin-top: 8%;">
+				<span >Here you can describe your week</span>
```
Comment (nastik1811) [inline lab3/index.html]: Also a heading

File: lab3/index.html [hunk 7]
```diff
@@ -0,0 +1,2406 @@
...
+
+	<footer>
+		<div class="footer">
+			<div class="footer-content">
+				<p class="copyright">Margarita Mazets © 2022</p>
+				<nav class="options">
+					<a href="https://github.com/rita-mazets"><img src="assets/images/github.png" alt="github logo" class="link-icon"></a>
+					<a href="https://t.me/mmmska_k"><img src="assets/images/telegram.png" alt="telegram logo" class="link-icon" style="margin-right: 45px;"></a>
+				</nav>
+			</div>
```
Comment (nastik1811) [inline lab3/index.html]: `nav` is usually used for main app navigation

File: lab3/logIn.html [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+				<main>
+					<div class="sign_in">
+						<span>Sign in with your email and password</span>
+						<form>
+							<div class="group">
+								<label class="form-input-label">Email</label>
+								<input class="form-input" name='email'type='email' required/>	
+							</div>
+							<div class="group">
+								<label class="form-input-label">Password</label>
```
Comment (nastik1811) [inline lab3/logIn.html]: labels should have `for` attribute

File: lab3/logIn.html [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+								<input class="form-input" name='password' type='password' required/>	
+							</div>
+							<span><a href="signUp.html" class="sign-mini-text">I don't have an account</a></span>
```
Comment (nastik1811) [inline lab3/logIn.html]: no need of wrapping it into `span`

File: lab3/signUp.html [hunk 10]
```diff
@@ -0,0 +1,199 @@
...
+										<option value="6">June </option>
+										<option value="7">July</option>
+										<option value="8">August </option>
+										<option value="9">September </option>
+										<option value="10">October </option>
+										<option value="11">November </option>
+										<option value="12">December </option>
+									</select>
+                                </div>
+                                <div  class="form-block-select" style="width: 30%; float:right;">
```
Comment (nastik1811) [inline lab3/signUp.html]: inline styles is evil 🙈

**Review by nastik1811 — CHANGES_REQUESTED** [review]
Looks good but needs some work on the html

File: lab3/index.html (hunk 4 above)
Comment (rita-mazets) [reply lab3/index.html]: It is anchor.

**Comment by nastik1811** [comment]
Why did you close this one?

**Comment by rita-mazets** [comment]
I fixed the bugs and opened a new one.

  
>Среда, 27 апреля 2022, 19:48 +03:00 от Anastasia Lugovskaya ***@***.***>:
> 
> 
>Why did you close this one?
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you modified the open/close state. Message ID:  <rita-mazets/MyCalendarSite/pull/2/c1111227553 @ github . com> 
 
 
С уважением,
Маргарита Мазец
***@***.***

**Comment by nastik1811** [comment]
There is no need to open a new PR when you've fixed something. You're just pushing changes to the same branch (PR will be updated automatically) and then re-request the review

**Comment by nastik1811** [comment]
If I'm not mistaken you still can re-open this PR, so we could continue discussion here 

P.S. it makes more sense, as I can see comments I left here and which one you have fixed or explained

**Comment by nastik1811** [comment]
And GitHub even gives me what comments are fixed.🙂 
Screenshot 2022-04-28 at 10 44 12

**Comment by rita-mazets** [comment]
I am sorry for the inconvenience, I opened everything.

**Review by nastik1811 — APPROVED** [review]
No worries )

For this work you get `9` as well

**Pull request merged**